clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/validators/BasicGoogleDeployDescriptionValidator.groovy

```diff
@@ -46,7 +46,9 @@ class BasicGoogleDeployDescriptionValidator extends DescriptionValidator<BasicGo
                                 description.credentials)
 
     if (description.minCpuPlatform) {
-      helper.validateMinCpuPlatform(description.minCpuPlatform, description.zone, description.credentials)
+      helper.validateMinCpuPlatform(description.minCpuPlatform,
+                                    description.regional ? description.region: description.zone,
```

Comment:
Missing a space just before the `:` here and below.

Reply:
Yep 2 for 2 :+1: